src/pretalx/static/cfp/js/proposalTabTitles.js

```diff
@@ -0,0 +1,23 @@
+const titleParts = document.title.split("::")
+
+document.addEventListener("DOMContentLoaded", () => {
```

Comment:
This part should probably go inside the `titleParts.length !== 3` safeguard too, just so that we don't touch the title at all if we don't know what's going on.